data-prepper-plugins/otel-trace-source/src/main/java/org/opensearch/dataprepper/plugins/source/oteltrace/OTelTraceSource.java

```diff
@@ -80,6 +83,12 @@ public OTelTraceSource(final OTelTraceSourceConfig oTelTraceSourceConfig, final
         this.pipelineName = pipelineDescription.getPipelineName();
         this.authenticationProvider = createAuthenticationProvider(pluginFactory);
         this.requestExceptionHandler = new GrpcRequestExceptionHandler(pluginMetrics);
+        this.byteDecoder = new OTelTraceDecoder();
+    }
+
+    @Override
+    public ByteDecoder getDecoder() {
```

Comment:
how is this used ?

Reply:
It is used during pipeline initialization in PipelineTransformer.java